src/main/scala/thurloe/service/ApiDataModels.scala

```diff
@@ -8,7 +9,22 @@ object ApiDataModelsJsonProtocol extends DefaultJsonProtocol {
   implicit val userKeyValuePairsFormat = jsonFormat2(UserKeyValuePairs)
 }
 
+object ThurloeQuery {
+  private def findParameters(wantedParameters: Seq[String], paramList: Seq[(String, String)]): Map[String, Option[String]] = {
+    val params2map = paramList.toMap
+    wantedParameters map { p => (p, params2map.get(p)) } toMap
+  }
+
+  def apply(params: Seq[(String, String)]): ThurloeQuery = {
+    val params2map = params.toMap
```

Comment:
This appears to silently and unpredictably drop query key/values where the key appears multiple times.  So

`thurloe?key=foo&key=bar` 

might query for `foo` or `bar` keys, but it's not clear which.

Cromwell's query API has an example of "and"ing multiple instances of the same query key together if you want to support that, or this could just error if you don't think that's worth dealing with.